components/ws-manager-api/typescript/src/core_grpc_pb.d.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
```

Comment:
I believe we should not be updating copyright year on existing files.

Reply:
> I believe we should not be updating copyright year on existing files.

That's 💯 correct.

That said, the auto-generated files will automatically update because they're autogenerated without headers, and the headers are added in afterwards. We could manually correct that, but chances are it will just happen the next time around.